app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
```

Comment:
Code Style: According to the checklist, it's better to use parentheses `()` when importing multiple classes/modules. Please, adjust your import statement accordingly.